apps/wfo-ui/pages/start-workflow/[workflowName].tsx

```diff
@@ -1,15 +1,20 @@
 import React from 'react';
 import { useRouter } from 'next/router';
+import { WFOStartWorkflowPage } from '@orchestrator-ui/orchestrator-ui-components';
 
 const StartWorkflowPage = () => {
     const router = useRouter();
-    const { workflowName: workflowNameQueryParameter } = router.query;
+    const { workflowName, productId } = router.query;
 
-    const processName = Array.isArray(workflowNameQueryParameter)
-        ? workflowNameQueryParameter[0]
-        : workflowNameQueryParameter;
-
-    return <h1>Start workflow: {processName}</h1>;
+    if (workflowName && productId) {
+        return (
+            <WFOStartWorkflowPage
+                workflowName={workflowName as string}
```

Comment:
The router is able to return an array, i think it is better to use the mechanism that is removed (Array.isArray) -- maybe hide it in a util file

Reply:
Anyway.. util functions is fine aswell. Will implement